credential-status/src/test/java/me/uport/sdk/credential_status/Tests.kt

```diff
@@ -0,0 +1,12 @@
+package me.uport.sdk.credential_status
+
+import org.junit.Test
+
+class Test {
+
+    @Test
+    fun `dummy test for jacoco buil report`() {
```

Comment:
ultimately, the `credential-status` module should be responsible for delegating the status checks to various status methods (similar to the way the universal resolver delegates based on DID method)

At that point, this class will probably have more to say :)